c/jsonschema.c

```diff
@@ -123,6 +123,7 @@ typedef struct PatternProperty_tag {
 
 typedef struct JSValueSpec_tag {
   int      typeMask;
+  int      type;
```

Comment:
this needs to be at the end of the struct to not wreck any plugin that depends upon the old struct.